cms/release_calendar/models.py

```diff
@@ -48,14 +48,20 @@ class ReleaseCalendarPage(BasePage):  # type: ignore[django-manager-missing]
     summary = RichTextField(features=settings.RICH_TEXT_BASIC)
 
     release_date = models.DateTimeField(
-        blank=True, null=True, help_text=_("Required once the release has been confirmed.")
+        blank=True,
+        null=True,
+        help_text=_("Required once the release has been confirmed."),
```

Comment:
How come these changed?. Is something wrong with our formatter setup? There shouldn't be a change between PRs unless some config has changed.

Reply:
Did some digging. I think it was my setup locally. As I have multiple formatters on VS Code. Black and Ruff.

My default formatter was set up as `Black` and it formatted this file for the line too long issues. I have made the default formatter to `ruff` now.

Now this file is formatted so it's consistent with the `cms/release_calendar/models.py` file.